Components/AccordionVertical/style.styl

```diff
@@ -0,0 +1,69 @@
+$rupture.scale = lookup('$globalRuptureScale') || (0 575px 767px 991px 1199px)
+$rupture.anti-overlap = lookup('$globalRuptureAntiOverlap') || -1px
+$containerPadding = lookup('$globalContainerPadding') || 15px
+$containerMaxWidth = lookup('$globalContainerMaxWidth') || 1140px
+$colorBrand = lookup('$globalColorBrand') || #254fc7
+$buttonBackgroundColor = #f3f3f3
```

Comment:
Thought it's the default flow for components to always lookup values.
But I agree maybe only the layout values should use lookups and colors in basecomponents should be defined without lookups first?